### PR TITLE
README init file

### PR DESCRIPTION
**Pull request by manuelperdigons**
Hi bro, I am creating a pull request in this project to add a README file that was missing for it, also to practice git flow and readme skills jajajaj

Tell me if you have a feedback of it, take care.

### CODE REVIEW

**Comment by manuelperdigons** [comment]
[](https://media.giphy.com/media/xUnPFR7qYiCvuIVhO4/giphy.gif)

File: README.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+## Testing
+
+* Run `npm run test` and it will run all test that are storaged on `/test` folder
+
+## Libraries / Dependencies
+
+- [Axios](https://github.com/axios/axios) - Used to fetch flights from APIs
+- [Dotenv](https://www.npmjs.com/package/dotenv) - Used to storage environment variables in a .env file 
+- [Express](https://expressjs.com/) - Used for structure of project
```
Comment (brahyam) [inline README.md]: I think Express is doing a little more than only giving structure to the project 😅

File: README.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# Flights Application
+
+Software application that is able to request all flight data from two APIs (<https://discovery-stub.comtravo.com/source1>) and (<https://discovery-stub.comtravo.com/source2>) and then it will send back to customer a response in JSON format of all flights but deleting duplicates in the process.
```
Comment (brahyam) [inline README.md]: "Software Application"

**Review by brahyam — APPROVED** [review]
It's very good, short, concise and has everything somebody would need to run it. In the future when you do something similar and need to put it in a repo for a company you could also add an "improvements" section, to say a few things on how to improve it or what would you do next, or what you didn't have enough time to do properly

File: README.md (hunk 1 above)
Comment (manuelperdigons) [reply README.md]: hahahaha what do you think I should write there?

File: README.md (hunk 2 above)
Comment (manuelperdigons) [reply README.md]: got it haha

Comment (brahyam) [reply README.md]: Hahaha nah, I was just thinking that it looks so formal and fancy. All good

File: README.md (hunk 1 above)
Comment (brahyam) [reply README.md]: I would say you're using it as web framework to serve the flights endpoint.

**Pull request merged**